### DISCUSSION TITLE
Plan for final submission

### DISCUSSION THREAD

**Discussion by DylanTh20** [ecn310/course-project-taxes-tariffs#16] [General]
| Task | Target Date | Completed? | Assignee | Notes |
|---|---|---|---|---|
| Working hypothesis w/data needs | Nov. 5 | done | Dylan, Kathryn, and Umar | Finalize our working hypothesis including theory, research question and causal diagram by looking it over one final time |
| Data | Nov. 10 | done | Dylan, Kathryn, Umar| First we need to convert both our datasets from wide to long, then cleanse missing observations, then merge the two datasets  |
| Preliminary replication package plan | Nov. 10 | done | Kathryn | Where to store this on repo?|
| First draft of data section of paper | Nov. 12 | done | Dylan and Umar | Pre-work in data.tex |
| First draft of data portion of replication package | Dec. 9 | | Dylan, Kathryn, and Umar | Pre-work in data.tex |
| Present preliminary analysis results in class for feedback | Nov. 14 | done | Dylan and Umar | |
| First draft of results / discussion section(s) | Nov. 14 | done | Dylan, Kathryn, and Umar | all figures/tables need to be done |
| Finalize replication package plan | Nov. 17 | done | Kathryn | |
| Revise analysis section | Dec. 14 | | Dylan and Umar | |
| Present analysis results w/theory in class for feedback | Nov. 19 | done | Dylan and Umar | |
| Present full project in class for feedback | Nov. 19 | done | Dylan and Umar | |
| Finalize analysis section | Dec. 13 | | Dylan and Umar | |
| Finalize data section | Dec. 13| | Dylan and Umar | |
| Finalize hypothesis/theory section | Dec. 13 | | Dylan | |
| Finalize lit review and references | Dec. 13 | | Kathryn | |
| Finalize discussion / conclusion | Dec. 13 | | Kathryn | |
| Finalize abstract / intro | Dec. 13 | | Umar | |
| Execute replication package plan | Dec. 12 | | Dylan, Kathryn, and Umar | |
| First draft of report | Dec. 13 | | Dylan, Kathryn, and Umar | |
| Revise replication package | Dec. 9 | | Dylan, Kathryn, and Umar | |
| Finalize replication package for testing | Dec. 7 | | Dylan, Kathryn, and Umar | |
| Revise report | Dec. 15 | | Dylan, Kathryn, and Umar | |
| Deliver report to outside reader | Dec. 3 | | Kathryn | Someone from class |
| Replication package testing by insider | Dec. 8 | | Umar | |
| Preliminary replication package testing by outsider | Dec. 13 | | Nadin Prabhu will test our replication package | |
| Edit report /  incorporate feedback from outsider reader | Dec. 15 | | Dylan, Kathryn, and Umar | |
| Final replication package testing by outsider | Dec. 14 | | Nadin Prabhu will test our replication package | |
| Final proofread of replication package | Dec. 15 | | Dylan, Kathryn, and Umar | Will use grammarly (or the writing center) |
| Final proofread of report | Dec. 15 | | Dylan, Kathryn, and Umar | Will use grammarly (or the writing center) |
| Submit whole package | Dec. 16 |   | Dylan, Kathryn, and Umar |  |

**Comment by Ksarrge** [comment]
What has been completed: 
- Getting help from Prof. in order to help reshape the data 10/29/2024

We are currently working on:
- Data reproducibility package Due: 11/01/2024
- Data.tex update to reflect new variables and if the variables are nominal or conditional Due: 11/01/2024
- Final submission plan Due: 11/01/2024

We need to work on in the future:
1. Run correlations, hypothesis tests, etc. ; confirm “story”
2. Write Results, Discussion sections
3. Write Theory / Hypothesis
4. Finalize Replication Package / Data Appendix
5. Expand and finalize Lit Review, References
6. Finalize Intro and Conclusion
7. Write Abstract

Final submission plan: 

Question for group: is the table needed? I feel like we can add revisions later but most of the tasks are listed here. Please advise otherwise I will not add a table since we do not need it. 

1. 11/01/2024 - Complete Stata do file and reproduciblity package - get feedback and adjust as nessecary - resubmit if nessecary 

2. 11/05/2024 - Complete the first correlation tests - confirm what tests we must carry out and how to do so in Stata 

3. 11/07/2024 - Create a discussion which explains our results and if they confirm our hypothesis 

Items 2 and 3 have a flexible deadline because we may need feedback or may make errors which will require the help of Professor Buzard 

4. 11/07/2024 - Write hypothesis and clear why theory (hard deadline) 

5. 11/14/2024 - Expand and finalize lit review to incorporate 10 sources; it has to be done by this date but we can add sources anytime before this date --> keep Lit review Log updated
